packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -416,6 +407,45 @@ paths:
     expect(response.get('Location')).match(expectedUrl);
   });
 
+  it('honors "x-forwarded-host" headers', async () => {
+    const app = new Application();
+    app.component(RestComponent);
+    const server = await app.getServer(RestServer);
+
+    const response = await createClientForHandler(server.requestHandler)
+      .get('/swagger-ui')
+      .set('x-forwarded-proto', 'http')
+      .set('x-forwarded-host', 'example.com:8080,my.example.com:9080');
+    await server.get(RestBindings.PORT);
+    const expectedUrl = new RegExp(
+      [
+        'https://loopback.io/api-explorer',
+        '\\?url=http://example.com:8080/openapi.json',
```

Comment:
Shouldn't this not include the `:8080`?

Reply:
It should for this case. For bluemix, I think you use `https://<your-bm-host>/...` with the default port `443` for https. The new code won't accidentally add `8080` as it's from the config, not the header.